util/sim/sim_utils.py

```diff
@@ -79,18 +80,21 @@ def resolve_relative_path(base_path, s):
 
 
 # Create simulation objects from a test list file
-def get_simulations(testlist, simulator):
+def get_simulations(testlists, simulator):
     # Get tests from test list file
-    testlist_path = Path(testlist).absolute()
-    with open(testlist_path, 'r') as f:
-        tests = yaml.safe_load(f)['runs']
-    # Convert relative paths in testlist file to absolute paths
-    for test in tests:
-        test['elf'] = testlist_path.parent / test['elf']
-        if 'cmd' in test:
-            test['cmd'] = [resolve_relative_path(testlist_path.parent, arg) for arg in test['cmd']]
+    all_tests = []
+    for testlist in testlists:
+        testlist_path = Path(testlist).absolute()
+        with open(testlist_path, 'r') as f:
+            tests = yaml.safe_load(f)['runs']
+        # Convert relative paths in testlist file to absolute paths
+        for test in tests:
+            test['elf'] = testlist_path.parent / test['elf']
+            if 'cmd' in test:
+                test['cmd'] = [resolve_relative_path(testlist_path.parent, arg) for arg in test['cmd']]
```

Comment:
break into lines? haha flake8 just complains...
```suggestion
                test['cmd'] = [resolve_relative_path(testlist_path.parent, arg)\
                 for arg in test['cmd']]
```

Reply:
Yeah chatgpt helped me a bit thanks!